packages.makefile

```diff
@@ -0,0 +1,98 @@
+#####################################################################################
+# Copyright 2011 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+include $(TOOLS_DIR)/build-package.makefile
+
+ARCHI := $(shell uname -m)
+OS := $(shell . SOURCES/rudder-sources/rudder-packages/detect_os.sh && echo $${OS})
+OSSP := $(shell . SOURCES/rudder-sources/rudder-packages/detect_os.sh && echo $${OSSP})
+OSVERSION := $(shell . SOURCES/rudder-sources/rudder-packages/detect_os.sh && echo $${OSVERSION})
```

Comment:
As agreed, we'll make sure to change the path here to ../detect_os.sh, once we have that file copied to all our builders.